production_start.py

```
@@ -0,0 +1,382 @@
+            logger.info("✅ Background services stopped")
+        
+        # Close database connections
+        await engine.dispose()
+        logger.info("✅ Database connections closed")
+        
+        # Close Redis connections
+        try:
+            redis = await get_redis_client()
+            await redis.close()
+            logger.info("✅ Redis connections closed")
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)